etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -0,0 +1,413 @@
+# compute_firewalld_rules: "{{ stackhpc_firewalld_rules }}" in compute.yml
+stackhpc_firewalld_rules: |
+  {% set stackhpc_firewalld_rules_verified = [] %}
+  {% for rule in stackhpc_firewalld_rules_unverified | unique %}
+  {% if rule | ansible.utils.remove_keys('state') in stackhpc_firewalld_rules_verified | map('ansible.utils.remove_keys', 'state') %}
+  {% set stackhpc_firewalld_rules_verified = 'Invalid configuration! Two matching firewalld rules exist with different states' + 1 %}
+  {% elif rule.network in network_interfaces and rule.network | net_zone is not none %}
+  {% set _ = stackhpc_firewalld_rules_verified.append(rule) %}
+  {% endif %}
+  {% endfor %}
+  {{ stackhpc_firewalld_rules_verified }}
+
+# Rules in this list may still contradict each other or reference non-existent
+# zones and interfaces
+stackhpc_firewalld_rules_unverified: |
+  {{ stackhpc_common_firewalld_rules +
+     (stackhpc_controller_firewalld_rules if 'controllers' in group_names else []) +
+     (stackhpc_compute_firewalld_rules if 'compute' in group_names else []) +
+     (stackhpc_storage_firewalld_rules if 'storage' in group_names else []) +
+     (stackhpc_monitoring_firewalld_rules if 'monitoring' in group_names else []) +
+     (stackhpc_seed_firewalld_rules if 'seed' in group_names else []) +
+     (stackhpc_seed_hypervisor_firewalld_rules if 'seed-hypervisor' in group_names else []) +
+     (stackhpc_wazuh_manager_infra_vm_firewalld_rules if 'wazuh-manager' in group_names else []) +
+     (stackhpc_ansible_control_infra_vm_firewalld_rules if inventory_hostname == 'localhost' else []) }}
+
+###############################################################################
+# Common firewalld rules
+
+stackhpc_common_firewalld_rules: "{{ stackhpc_common_firewalld_rules_default + stackhpc_common_firewalld_rules_extra }}"
+
+stackhpc_common_firewalld_rules_default: |
+  {% set stackhpc_common_firewalld_rules_formatted = [] %}
+  {% for ruleset in stackhpc_common_firewalld_rules_template %}
+  {% if ruleset.enabled %}
+  {% for rule in ruleset.rules %}
+  {% set _ = stackhpc_common_firewalld_rules_formatted.append(rule | combine({'zone': rule.network | net_zone })) %}
+  {% endfor %}
+  {% endif %}
+  {% endfor %}
+  {{ stackhpc_common_firewalld_rules_formatted }}
+
+stackhpc_common_firewalld_rules_template:
+  # Common
+  - rules:
+      - service: ssh
+        network: "{{ admin_oc_net_name }}"
+        state: enabled
+    enabled: true
+  - rules:
+      - service: cockpit
+        network: "{{ public_net_name }}"
+        state: disabled
+      - service: dhcpv6-client
+        network: "{{ public_net_name }}"
+        state: disabled
+    enabled: "{{ 'public' in stackhpc_firewalld_zones }}"
+  - rules:
+      - service: ssh
+        network: "{{ public_net_name }}"
+        state: disabled
+    enabled: "{{ 'public' in stackhpc_firewalld_zones && admin_oc_net_name | net_zone != 'public' }}"
```

Comment:
```suggestion
    enabled: "{{ 'public' in stackhpc_firewalld_zones and admin_oc_net_name | net_zone != 'public' }}"
```

Reply:
And do this elsewhere as well

Reply:
Done. Trying to get out of that bad habit